src/open_inwoner/js/components/siteimprove/tracking.js

```diff
@@ -11,25 +11,29 @@
 //     },
 //   }
 // }
-//
-// if (typeof _sz !== 'undefined') {
-//   console.log('-> SiteImprove _sz object exists: ', _sz)
-// } else {
-//   console.log('-> SiteImprove _sz is not defined yet.')
-// }
 
-// Main script file
+if (typeof _sz !== 'undefined') {
+  console.log('-> SiteImprove _sz object exists: ', _sz)
+} else {
+  console.log('-> SiteImprove _sz is not defined yet.')
+}
```

Comment:
Currently we have these logs in both Develop and Production due to the ongoing Siteimprove issues, so until https://taiga.maykinmedia.nl/project/open-inwoner/issue/2622 is solved - we still need these logs because sometimes we tend to do 'hacky' things on Production🙈